app/Platform/Actions/BuildGameListAction.php

```diff
@@ -156,6 +157,16 @@ private function buildBaseQuery(GameListType $listType, ?User $user = null): Bui
         }
 
         switch ($listType) {
+            case GameListType::AllGames:
+                // Exclude events, hubs, subsets, and inactive systems.
+                $query->where('GameData.ConsoleID', '!=', System::Events)
+                    ->where('GameData.ConsoleID', '!=', System::Hubs)
+                    ->where('GameData.Title', 'not like', "%[Subset -%")
+                    ->whereHas('system', function ($q) {
+                        return $q->active();
+                    });
```

Comment:
Recommend replacing the ConsoleID checks with `System::scopeGameSystems`.

(untested):
```
->whereHas('system', function ($q) {
    return $q->gameSystems()->active();
});
```

Reply:
That seems to work great. Applied in latest.